jest.config.ts

```diff
@@ -1,3 +1,4 @@
+import { readFileSync } from 'node:fs';
 import os from 'node:os';
 import v8 from 'node:v8';
 import type { InitialOptionsTsJest } from 'ts-jest/dist/types';
```

Comment:
whoops, that import is wrong, can break in future. but can be fixed in separate PR.

Reply:
Do you mean `node:os` and `node:v8`too?